Rocket.Chat/Controllers/Chat/ChannelActionsViewController.swift

```diff
@@ -30,6 +30,7 @@ class ChannelActionsViewController: BaseViewController {
             guard let subscription = self.subscription else { return }
 
             let shouldListMentions = subscription.type != .directMessage
+            let shouldListMembers = subscription.type != .directMessage
```

Comment:
We could replace `shouldListMentions` and `shouldListMembers` for just one flag called `isDirectMessage`